BoxContentSDK/BoxContentSDK/Requests/BOXRecentItemsRequest.m

```diff
@@ -41,7 +41,7 @@ - (BOXAPIOperation *)createOperation
     }
 
     if (self.nextMarker != nil) {
-        queryParameters[BOXAPIParameterKeyMarker] = self.nextMarker;
```

Comment:
Should we mark the `BOXAPIParameterKeyMarker` key as deprecated too to prevent confusion?